.gitignore

```
@@ -0,0 +1,4 @@
+node_modules/
+dist/main.js
+dist/main.js.LICENSE.txt
+.ds_store
```

.DS_Storeがまだ消えてないようです。
```suggestion
.DS_Store
```

---

.DS_Store
が虫眼鏡マークを使っても出てきません💦

---

すみません、虫眼鏡マークとは何ですか？